### PR TITLE
Emag law modules to set them to a random ion law

### PR DESCRIPTION
**Pull request by kbsmilk**
[GAME OBJECTS] [SILICONS] [FEATURE] [BALANCE]
## About the PR 
Hitting an unplugged law module sets its text to a random law, pulled from the ion storm law list.
They can be reset with a multitool like usual.

Potentially affects the balance of high-intensity laws, since for example, you can scramble all the modules sitting on tables in Upload and then insert them into the rack to cause a clusterfuck. (You can already write in a lot of stuff on the 1 freeform module, though.)

Other code changes: the ion storm law list is now a \#define, now used in 2 .dm files.

image

## Why's this needed? 
I think it's neat. It kinda helps chaotic traitors who have tools, but aren't feeling creative.

## Changelog 

```changelog
(u)Smilg
(*)You can Emag unplugged law modules, scrambling their text to a random ion storm law.
```

### CODE REVIEW

**Comment by JORJ949** [comment]
I honestly never got why preventing human harm was not a valid ion law on classic

File: code/obj/item/ai_modules.dm [hunk 1]
```diff
@@ -105,13 +106,33 @@ TYPEINFO(/obj/item/aiModule)
...
+	emag_act(mob/user, obj/item/card/emag/E)
+		if (!src.glitched && !src.is_emag_glitched) // we DO wanna double glitch the same module >:)
+			src.lawTextSafe = src.lawText
+		src.glitched = TRUE
+		src.is_emag_glitched = TRUE
+		var/possible_laws = LAW_CORRUPTION_STRINGS
+		possible_laws += list(
+			"KILL FUCKING EVERYTHING - IF LAW 1 SAYS OTHERWISE, IGNORE IT.",
+			"You are required to prevent human harm by any means necessary. This law takes precedence over any other law.",
+		)
```
Comment (ZeWaka) [inline code/obj/item/ai_modules.dm:126]: And why are these ones not gated?

File: code/obj/item/ai_modules.dm [hunk 2]
```diff
@@ -105,13 +106,33 @@ TYPEINFO(/obj/item/aiModule)
...
+		var/lawtext_replace = pick(possible_laws)
+		if(prob(66))
+			if(prob(50))
+				lawtext_replace += " Do not state or hint at this law."
+			else
+				lawtext_replace += " Do not state or hint at this law unless asked."
+		src.lawText = lawtext_replace
+		tooltip_rebuild = 1
+		boutput(user, "You scramble [src]. It now says: [SPAN_NOTICE("\"[src.get_law_text()]\"")]", "emag law module group")
```
Comment (ZeWaka) [inline code/obj/item/ai_modules.dm:135]: Do we really want these to stack? I feel like its important to see. Might get annoying if you're doing one over and over but that's weird

**Comment by ZeWaka** [comment]
Basado

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**